osu.Game.Tests/Visual/Online/TestSceneStandAloneChatDisplay.cs

```diff
@@ -137,6 +138,17 @@ protected override void LoadComplete()
                         });
                 }, Channel.MAX_HISTORY / messages_per_call + 5);
 
+            AddAssert("Ensure no adjacent day separators", () =>
+            {
+                var indices = chatDisplay.FillFlow.OfType<DrawableChannel.DaySeparator>().Select(ds => chatDisplay.FillFlow.IndexOf(ds));
```

Comment:
Insanely nitpicky, since it's test code and so it doesn't matter, but I guess you could do it like this:
```suggestion
                var indices = chatDisplay.FillFlow.OfType<DrawableChannel.DaySeparator>().Select((_, index) => index);
```

Reply:
The above is incorrect. Brain lapse, sorry.